lambdaclass/curse_of_myrra
# Poisoned Feedback

Closes #600 
This PR implements a feedback for the poisoned status in the three currently available characters.

How to test:
1. Start a match, where at least one of the playing characters is Uma.
2. Go near to a rival and use the poison skill.
3. You should notice a cloud of green smoke that represents the poisoned status. It should go off after 5 seconds.
